### PR TITLE
Make C to c in the word Connects

### PR DESCRIPTION
**Pull request by Abbhiishek**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/789"></a>

### CODE REVIEW

**Comment by Panquesito7** [comment]
I'm not sure if this should be made. I think it is OK as-is. Let's see what other people think. 🙂

**Comment by shreyaansjain06** [comment]
I think we dont need this change

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # LinkFree by EddieHub
 
-- LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
+- LinkFree Connect audiences to all of your Content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
```
Comment (krishguptadev) [inline README.md:3]: I think the issue was 👇 

```suggestion
- LinkFree connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
I suggested to also lowercase `connect`

**Comment by Abbhiishek** [comment]
> I suggested to also lowercase `connect`

ohh ! I see there was no word as connects.🙂
cool !

**Review by Vyvy-vi — COMMENTED** [review]
lgtm... Though, we can probably use "connects" over here, in order to be grammatically correct

**Review by schmelto — APPROVED** [review]

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
It should be ‘connects’ with an s on the end.

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
Looks good! Thanks for contributing!

**Pull request merged**